mirrulations-client/src/mirrclient/client.py

```diff
@@ -354,10 +354,8 @@ def perform_attachment_job(self, url):
         file_info = response_data["attributes"]["fileFormats"]
         file_urls, file_types = get_urls_and_formats(file_info)
 
-        attachments = download_attachments(
-            file_urls, file_types, self.client_id)
-
-        return attachments
+        return download_attachments(
+            file_urls, file_types, job_id)
```

Comment:
Changed download_attachments here to use job_id instead of client_id, and added the job_id parameter to perform_attachment_job(). I had to return the results directly because the function was at the limit of local variables and that was the easiest fix